docs/how-to-guides/gitops-using-azure-arc-onboarding-component.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Deploy configuration using GitOps on an Azure Arc enabled Lokomotive cluster
+weight: 10
+---
+
+## Introduction
+
+Azure Arc offers simplified management, faster app development, and consistent Azure services.
+
+With Azure Arc, you can:
+
+- Centrally manage a wide range of resources including
+[Windows](https://azure.microsoft.com/en-in/campaigns/windows-server/) and
+[Linux](https://azure.microsoft.com/en-in/overview/linux-on-azure/) servers, SQL server,
+[Kubernetes](https://azure.microsoft.com/en-in/services/kubernetes-service/) clusters and [Azure
+services](https://azure.microsoft.com/en-in/services/azure-arc/hybrid-data-services/).
+
+- Establish central visibility in the [Azure portal](https://azure.microsoft.com/en-in/features/azure-portal/)
+and enable multi-environment search with Azure Resource Graph.
+
+- Meet [governance](https://azure.microsoft.com/en-in/solutions/governance/) and compliance standards for
+apps, infrastructure and data with [Azure Policy](https://azure.microsoft.com/en-in/services/azure-policy/).
+
+- Delegate access and manage security policies for resources using role-based access control (RBAC) and [Azure
+Lighthouse](https://azure.microsoft.com/en-in/services/azure-lighthouse/).
+
+- Organise and inventory assets through a variety of Azure scopes, such as management groups, subscriptions,
+resource groups and tags.
+
+This guide provides the steps for onboarding a Lokomotive cluster on Azure Arc.
+
+## Learning objectives
+
+At the end of this tutorial you will have onboarded a Lokomotive cluster on Azure Arc, create an Azure Arc
```

Comment:
```suggestion
At the end of this tutorial, you will have onboarded a Lokomotive cluster on Azure Arc, created an Azure Arc
```

Reply:
rephrased to 
```
By the end of this guide, the following things would be accomplished:
* Onboard a Lokomotive cluster on Azure Arc
* Create Azure Arc Kubernetes configuration for the GitOps agent.
* Watch the GitOps agent deploy the deployments in the git repository.
```

Let me know what you think ?